From 98822d549689f09bd454b1cf462aae231931f64d Mon Sep 17 00:00:00 2001
From: mancha <mancha1 AT zoho DOT com>
Date: Sat, 14 Feb 2015
Subject: Fix compile with FreeType 2.5.1+

FREETYPE_INCLUDE_DIR_freetype2 & FREETYPE_INCLUDE_DIR_ft2build 
might differ. Include both.

---
 CMakeLists.txt |    1 +
 1 file changed, 1 insertion(+)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -171,6 +171,7 @@ include_directories(
 	${X11_Xrender_INCLUDE_PATH}
 	${X11_Xrandr_INCLUDE_PATH}
 	${FREETYPE_INCLUDE_DIR_freetype2}
+	${FREETYPE_INCLUDE_DIR_ft2build}
 	${X11_Xmu_INCLUDE_PATH}
 	${ZLIB_INCLUDE_DIR}
 	${JPEG_INCLUDE_DIR}
